docs/deploy_route53.md

```diff
@@ -1,34 +1,44 @@
 # AWS based deployment with Route53 integration
 
-Here we provide an example of k8gb deployment in AWS context with Route53 as edgeDNS provider
+Here we provide an example of k8gb deployment in AWS context with Route53 as edgeDNS provider.
 
-## Reference setup
+## Reference Setup
 
 Two EKS clusters in `eu-west-1` and `us-east-1`.
 
 Terraform code for cluster reference setup can be found [here](https://github.com/AbsaOSS/k8gb/tree/master/docs/examples/route53)
 
 Feel free to reuse this code fully or partially and adapt for your existing scenario
-things like IRSA(IAM Roles for Service Accounts)
+things like IRSA(IAM Roles for Service Accounts).
+
+## Install Ingress Controller
+
+```sh
+kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/controller-v0.40.2/deploy/static/provider/aws/deploy.yaml
+```
 
 ## Deploy k8gb
 
-Example `values.yaml` override configs can be found [here](https://github.com/AbsaOSS/k8gb/tree/master/docs/examples/route53/k8gb)
+Example helm configuration files can be found [here](https://github.com/AbsaOSS/k8gb/tree/master/docs/examples/route53/k8gb)
+
+Modify them to reflect your `dnsZone`, `edgeDNSZone`, valid `hostedZoneID` and `irsaRole` ARN.
 
-You can use `helm` to deploy stable release from Helm repo
+Clone k8gb repository and use `helm` with custom values
 
 ```sh
+git clone https://github.com/AbsaOSS/k8gb.git
+cd k8gb
+
 helm repo add k8gb https://www.k8gb.io
-```
+helm repo update
 
-Alternatively, use make target to deploy right from the git repository
+kubectl create ns k8gb
```

Comment:
just in case, helm v3 has `--create-namespace` option.
Also probably should be repeated for both `eu-west-1` and `us-east-1` clusters
```suggestion
```